src/styles/footer.css

```diff
@@ -0,0 +1,168 @@
+.footer-content {
+  width: auto;
+  margin: 10px auto;
+}
+
+.footer__logo {
+  margin: 0;
+  padding: 0;
+  width: 0px;
+  height: 0px;
+}
+
+.footer__bottomline {
+  width: auto;
+  height: 3px;
+  margin: 0 auto;
+  border-radius: 50%;
+  background-color: var(--primary-color);
+  background-image: linear-gradient(to right, var(--primary-color), var(--secondary-color));
+}
+
+.footer-credits {
+  text-align: center;
+  margin: 5px auto;
+  color: lightslategray;
+}
+
+.footer-link-wrapper {
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px auto;
+}
+
+.footer__icon {
+  width: 45px;
+  height: 45px;
+  border-style: solid;
+  border-color: var(--secondary-color);
+  border-radius: 50%;
+  padding: 10px;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-link__logo {
+  color: var(--secondary-color);
+  text-decoration: none;
+}
+
+.footer-link  {
+  color: var(--secondary-color);
+  text-decoration: none;
+}
+
+.footer-link:hover {
+  color: var(--primary-color);
+  text-decoration: none;
+}
+
+@media (min-width: 576px) {
+  .footer-content {
+    width: auto;
+  }
+
+  .footer__logo{
+    width: 0px;
+    height: 0px;
```

Comment:
If you won't display this element use `display: none` to hide it. Also this will be fix wrong left `margin` here:
![image](https://user-images.githubusercontent.com/36460224/85726510-70dd9000-b6f6-11ea-9070-405f826cbeee.png)